systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -242,4 +246,86 @@ public void testStrimziPodSetsFeatureGate(ExtensionContext extensionContext) {
         ClientUtils.waitForClientSuccess(producerName, INFRA_NAMESPACE, MESSAGE_COUNT);
         ClientUtils.waitForClientSuccess(consumerName, INFRA_NAMESPACE, MESSAGE_COUNT);
     }
+
+    @IsolatedTest
+    void testSwitchingStrimziPodSetFeatureGateOnAndOff(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String producerName = "producer-test-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String consumerName = "consumer-test-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicName = KafkaTopicUtils.generateRandomNameOfTopic();
+
+        int zkReplicas = 3;
+        int kafkaReplicas = 3;
+
+        final LabelSelector zkSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.zookeeperStatefulSetName(clusterName));
+        final LabelSelector kafkaSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.kafkaStatefulSetName(clusterName));
+
+        int messageCount = 500;
+
+        LOGGER.info("Deploying CO with STS - SPS is disabled");
+
+        clusterOperator.unInstall();
+        clusterOperator = new SetupClusterOperator.SetupClusterOperatorBuilder()
+            .withExtensionContext(BeforeAllOnce.getSharedExtensionContext())
+            .withNamespace(INFRA_NAMESPACE)
+            .withWatchingNamespaces(Constants.WATCH_ALL_NAMESPACES)
+            .createInstallation()
+            .runInstallation();
```

Comment:
To make this more future proof, you should expliciely disable the FG here.

Reply:
Makes sense, thanks.